fedimint-core/src/task.rs

```diff
@@ -617,6 +617,19 @@ impl<T: Sync> MaybeSync for T {}
 #[cfg(target_family = "wasm")]
 impl<T> MaybeSync for T {}
 
+// Used in tests when sleep functionality is desired so it can be logged.
+// Must include comment describing the reason for sleeping.
+pub async fn sleep_in_test(comment: impl AsRef<str>, duration: Duration) {
```

Comment:
might be to use `tracing::Span` instead of str comment

Reply:
Shouldn't `comment` be `reason`, and just another value with a fixed `"Sleep"` message? Or are there any other benefits? I'm not even sure how to do `Span` :D

Reply:
Span just lets you express variables better instead of using format!. also spans have parent-child relation.

something like this 
```
async move {
    info!(target: LOG_TEST, ?duration, "sleeping");
    sleep(duration).await;
}.instrument(span).await
```

Reply:
caller is expected to use `info_span!` or `debug_span!` to create spans

Reply:
Is it useful to create a new span just for a sleep? I guess if you already have one it's nice, but what would one gain from adding it here otherwise?